backend/backend.go

```diff
@@ -921,6 +921,7 @@ func (backend *Backend) Register(theDevice device.Interface) error {
 		backend.onDeviceUninit(theDevice.Identifier())
 		return err
 	}
+	theDevice.Observe(backend.Notify)
```

Comment:
there are lots of lines like this in backend.go, where in theory we should store the result (which is the unsubscribe function), and unsubscribe from observing once the instance the device/base/whatever is not available anymore.

Would do this in a different PR at some point, but not very high prio, as an unplugged device does not send any events.

Reply:
I guess you could call it a small memory leak?

Reply:
It is also a potential source of future bugs, if we add for example notifications that leak (e.g. by a long running goroutine that is not shut down or finishes after the device is unplugged). :shrug:

Reply:
I hope we have an issue for this so we won't forget!